resources/quarkus/launch.json

```diff
@@ -0,0 +1,18 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+		{
+            "name": "Run Camel Quarkus JVM application and attach Camel debugger",
+            "type": "apache.camel",
+            "request": "attach",
+            "preLaunchTask": "Start Camel application with Maven Quarkus Dev with camel.debug profile"
+        },
+        {
+            "name": "Run Camel native application and attach Camel debugger",
+            "type": "apache.camel",
+            "request": "attach",
+            "preLaunchTask": "Start Camel native application debug-ready"
```

Comment:
does it make sense to provide the launch configuration with attach debugger if we push for not having the extension pack? (the launch configuration won't work without the VS Code debug adapter for Camel)
Or maybe we need to find a way to mention that the other extension is mandatory

Reply:
good point.. I will remove .vscode creation